scripts/run-pr.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ -z "$1" ]; then
+  echo "No pull request ID was specified."
+  exit 1
+fi
+
+git fetch origin pull/${1}/head
```

Comment:
`git fetch https://github.com/thelounge/lounge.git refs/pull/${1}/head` would probably be better as people probably don't have `thelounge/lounge` as `origin`.